projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cdc/b2b/b2b-manage-users-e2e-spec.ts

```diff
@@ -0,0 +1,59 @@
+import { loginUser } from '../../../../helpers/checkout-flow';
+import { cdcB2BDelegateAdminUser } from '../../../../helpers/vendor/cdc/cdc';
+import * as alerts from '../../../../helpers/global-message';
+
+describe('Manage Users in CDC B2B scenario', () => {
+  beforeEach(() => {
+    cy.visit('/powertools-spa/en/USD/login');
+    loginUser(cdcB2BDelegateAdminUser);
+    cy.wait(3000);
```

Comment:
Waiting a hardcoded amount of time (e.g. 3000ms) is not recommended. Instead we should await e.g. specific the background http request.

Here's an example in Spartacus codebase: [usage](https://github.com/SAP/spartacus/blob/3fe8b68c0899d822a983879abc95fa5ee28d351c/projects/storefrontapp-e2e-cypress/cypress/integration/vendor/epd-visualization/visualization-lookup/visualization-lookup.e2e-spec.ts#L26) and [definition](https://github.com/SAP/spartacus/blob/3fe8b68c0899d822a983879abc95fa5ee28d351c/projects/storefrontapp-e2e-cypress/cypress/integration/vendor/epd-visualization/visualization-lookup/visualization-lookup.e2e-spec.ts#L8-L13) of " Cypress route alias".
  
For more, see:
- Cypress best practices for 'wait': https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting
- Cypress docs for 'wait: https://docs.cypress.io/api/commands/wait

--
This remark applies also to other places in this file

Reply:
done